core/src/scene/sceneLoader.cpp

```diff
@@ -64,7 +64,11 @@ std::string parseSequence(const Node& node) {
         try {
             sstream << val.as<float>() << ",";
         } catch (const BadConversion& e) {
-            logMsg("Error: Float value was expected for styleParam sequence value\n");
+            try {
+                sstream << val.as<std::string>() << ",";
+            } catch (const BadConversion& e) {
+                logMsg("Error: Float or Unit expected for styleParam sequence value\n");
```

Comment:
I think any yaml scalar value can be converted to string, so not sure if it makes sense to have a try catch for `val.as<std::string>()`